doc/release-notes/5.3-release-notes.md

```diff
@@ -44,7 +54,28 @@ The functionality previously provided by the DB settings :StatusMessageHeader an
 
 ### New Database Settings and JVM Options
 
-No new DB Settings or JVM Options have been added in this release. See above note about obsolete options.
+Several new DB options have been added in this release:
```

Comment:
"DB options" might be mistaken for "database-based settings" if not put into more context.

Reply:
clarified these are JVM options in https://github.com/IQSS/dataverse/commit/9b875614cf006e0d7b182e4aca8f7ddaf5fd3a57